src/components/input-group/input-group.scss

```diff
@@ -0,0 +1,28 @@
+@use "../../compiled/tokens/scss/size";
+
+.c-input-group {
+  display: flex;
+
+  & :first-child {
+    border-bottom-right-radius: var(--size-border-radius-input-group);
+    border-top-right-radius: var(--size-border-radius-input-group);
+  }
+
+  & :last-child {
+    border-bottom-left-radius: var(--size-border-radius-input-group);
+    border-top-left-radius: var(--size-border-radius-input-group);
+  }
+
+  & :nth-child(n + 2) {
+    margin-left: -2px;
+  }
+
+  /**
+ * In the instance where someone would want to include 3 or more elements,
+ * this ensures that the middle elements will not have a border radius.
+ */
+
+  & :not(:first-child):not(:last-child) {
+    border-radius: 0;
+  }
```

Comment:
Since buttons and inputs already have border-radius, it's probably possible to do something like this:

```css
& > :not(:first-child) {
  border-bottom-left-radius: 0;
  border-top-left-radius: 0;
}

& > :not(:last-child) {
  border-bottom-left-radius: 0;
  border-top-left-radius: 0;
}
```

This would replace this block but also the other border radii declarations.

Reply:
When I pasted this into what I already have it did this:

<img width="696" alt="Screen Shot 2021-07-16 at 3 01 10 PM" src="https://user-images.githubusercontent.com/24928337/126012621-1df94f20-28af-4578-a754-ca0abde02f14.png">

Reply:
I changed `& > :not(:last-child)` to right instead left and that seemed to do it